### ISSUE TITLE
New namespace request: mas

### ISSUE THREAD

**Issue by durera** [ansible/galaxy#2758]
## Namespace Request
Namespace: mas

##### Description
IBM Maximo Application Suite

##### GitHub Org Link
https://github.com/ibm-mas

##### Admins
- durera
- whitfiea
- tomklapiscak

**Comment by thedoubl3j** [comment]
greetings @durera since Maximo Application Suite is an IBM product/entity, we would suggest adding you all as admins on the IBM namespace rather than create a separate namespace.

**Comment by durera** [comment]
Yeah, that also works, mas was the namespace we've been using internally developing this, we can easily change that & move into the existing IBM namespace.  Do I need to open a seperate issue for that/find someone specific to approve that who is already an owner in that namespace, or is this issue sufficient to move forward?

**Comment by thedoubl3j** [comment]
I can add you directly to it @durera

**Comment by durera** [comment]
Confirming I have access and have published first release to verify everything work, thanks for the help.  

It's not the end of the world if it's not possible, but if it is could https://galaxy.ansible.com/durera/devops be cleaned up/nuked from the system?  I used my own account as a place to experiment with Galaxy as this is all new to us, & would be great to clear that out if possible incase anyone lands on that one by mistake when searching for the new collection.

**Comment by thedoubl3j** [comment]
we can't delete it but you can deprecate it @durera which will remove it from search results

**Comment by durera** [comment]
Thanks for the tip 👍
